modules/ossm-threescale-webassembly-module-examples-for-credentials-use-cases.adoc

```diff
@@ -0,0 +1,224 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-examples-for-credentials-use-cases_{context}"]
+= The 3scale WebAssembly module examples for `credentials` use cases
+
+You will spend most of your time applying configuration steps to obtain credentials in the requests to your services.
+
+The following are `credentials` example, which you can modify to adapt to specific use cases.
+
+You can combine them all, although when you specify multiple source objects with their own `lookup queries`, they are evaluated in order until one of them successfully resolves.
+
+.API key (`user_key`) in query string parameters
```

Comment:
I think all of these should be `==` H2s
Usually the `.headings` like this are reserved for certain uses, e.g. examples, prerequisites in modules, etc